src/footer/Footer-with-map.css

```diff
@@ -0,0 +1,134 @@
+.myFooter {
+    background-color: #182c39;
+    color: white;
+    padding-top: 20px;
+}
+
+.map-container {
+    height: 240px;
+    width: 100%;
+    margin-top: 30px;
+    margin-bottom: 10px;
+}
+
+.myFooter .row {
+    margin: 0 85px;
+}
+
+.myFooter .footer-copyright {
+    background-color: #10222e;
+    padding-top: 3px;
+    padding-bottom: 3px;
+    text-align: center;
+}
+
+.myFooter .footer-copyright p {
+    margin: 10px;
+    color: #ccc;
+}
+
+.myFooter .container {
+    width: auto;
+}
+
+.myFooter ul {
+    list-style-type: none;
+    padding-left: 0;
+    line-height: 1.7;
+}
+
+.myFooter h5 {
+    font-size: 18px;
+    color: white;
+    font-weight: bold;
+    margin-top: 30px;
+}
+
+.myFooter a {
```

Comment:
I suggest not to use a tags at all. You can use react native component. (possible it can be wrapper around the a tag)